packages/validator/src/commands/utils.ts

```diff
@@ -92,21 +93,36 @@ export function getConversionPath(
   }
 
   if (!fromVersion) {
-    let fromLatest;
+    let fromLatest, latestOverall;
+    const versionRegex = /^[0-9\.]+$/;
 
     for (const version in conversions[from]) {
-      if (!conversions[from][version][to]) {
+      latestOverall =
```

Comment:
You can compute `latestOverall` with the `sort` function on the `conversions[from]` keys, I'd say that would be more readable.